ecs-agent/netlib/model/networkinterface/networkinterface.go

```diff
@@ -431,19 +455,24 @@ func InterfaceFromACS(acsENI *ecsacs.ElasticNetworkInterface) (*NetworkInterface
 
 // ValidateENI validates the NetworkInterface information sent from ACS.
 func ValidateENI(acsENI *ecsacs.ElasticNetworkInterface) error {
-	// At least one IPv4 address should be associated with the NetworkInterface.
-	if len(acsENI.Ipv4Addresses) < 1 {
-		return errors.Errorf("eni message validation: no ipv4 addresses in the message")
-	}
+	ipv6OnlyTask := len(acsENI.Ipv6Addresses) > 0 && len(acsENI.Ipv4Addresses) == 0
```

Comment:
Why not call the `IPv6Only` method here?

Reply:
We are validating the payload here (the `acsENI` doesn't have that method)

Reply:
oh ok